### DISCUSSION TITLE
How do I hack with the wills cheat menu?

### DISCUSSION THREAD

**Discussion by CHAKAMAN12** [Prodigy-Hacking/ProdigyMathGameHacking#1308] [Questions] [answered]
How do i use it, i have learned how to hack other things, but not this one.

**Comment by afkvido** [comment, marked as answer]
Hi @CHAKAMAN12! To use Will's Cheat Menu, follow the steps:

1. [Get PHEx from the Chrome Web Store](https://chrome.google.com/webstore/detail/prodigyhacking/gjabpajagbgoifbkflgojeojmnlmioea)
2. Visit [play.prodigygame.com](https://play.prodigygame.com/)
3. Make a second account and log in. (DO NOT USE HACKS ON YOUR PRIMARY ACCOUNT). Hacks can damage your account if used improperly or excessively, and most hacks can't be undone.
4. Once you enter a world, A dropdown arrow will appear in the top left after 15-20 seconds. 
5. Click the dropdown arrow, all the hacks are there.

Video tutorial: https://youtu.be/OhfHNatEtw4


_____

If your hacks just say "Success" but nothing happens, [clear your cache and reload prodigy.](https://gist.github.com/afkvido/122d00d787003950210500f8fd9d3e7c)

**Reply by donaldli2020** [reply]
I was gonna say that the server is now up

**Reply by BellaBattilochi** [reply]
Can anyone make a hacked acc for me my email is boileghlila@gmail.com

**Reply by BellaBattilochi** [reply]
And have it done by today??

**Reply by ghost** [reply]
You can get it done today with this: https://prodigy-dashboard.hostedposted.com

**Comment by BellaBattilochi** [comment]
The problem is I don’t own a computer and I’m on school iPads so anyway u
guys could do it for me ?? Like make me a new account

**Reply by ghost** [reply]
just the the dashboard thing

**Reply by ghost** [reply]
computer & mobile

**Comment by BellaBattilochi** [comment]
Can u do it for me pls I tried it all day today and I can’t

**Reply by ghost** [reply]
Please tell what you want.

**Reply by 00100000** [reply]
yo @1603191909, please see https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/issues/1330#issuecomment-1038594373
